composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "imagick/imagick",
+    "type": "php-ext",
+    "license": "PHP-3.01",
+    "description": "The Imagick PHP extension",
+    "require": {
+        "php": ">= 5.4.0"
```

Comment:
Just looking to the Changelog notes and it was already supposed to be dropped after 3.4.4RC1 code still support 5.x (haven't tried to compile it). Might be potentially good to drop it though. Anyway it's probably ok to leave it for now and we can just then officially bump it.